tools/bin/commands/util/operator_funcs

```diff
@@ -36,3 +36,75 @@ check_operator_requirements() {
 
     echo "OK"
 }
+
+default_binary_path() {
+    local abs_path="$( cd "$( dirname "${BASH_SOURCE[0]}" )/../binaries" >/dev/null 2>&1 && pwd )"
+
+    if [[ -z "$abs_path" ]] ; then
+      # error; for some reason, the path is not accessible
+      # to the script (e.g. permissions re-evaled after suid)
+      exit 1  # fail
+    fi
+
+    echo ${abs_path}
+}
+
+# check that a given binary is in $PATH
+check_for_command() {
+    local cmd=${1:-curl}
+
+    if $(which ${cmd} >/dev/null 2>&1); then
+        return 0
+    else
+        return 1
+    fi
+}
+
+# check that the operator binary is in place
+check_operator_binary() {
+    local location=${1}
+    local binary=${2}
+
+    if [[ -x ${location}/${binary} ]]; then
+        if [[ $(${location}/${binary} -h | grep 'syndesis') ]]; then
+            return 0
+        fi
+    fi
+    return 1
+}
+
+# download the operator binary
+download_operator_binary() {
+    local location=${1:-"$(default_binary_path)"}
+    local binary=${2:-"syndesis"}
+    local base_url="https://github.com/syndesisio/syndesis/releases/download/1.8.1-20191004/syndesis-1.8.1-20191004-"
```

Comment:
You might want to `curl` it: `curl -s -L -H 'Accept: application/json' https://github.com/syndesisio/syndesis/releases/latest |jq .tag_name`

Reply:
I cant do that because this is a one time release, the binaries arent part of the release

Reply:
Now with releases containing all files, I can improve this bit. I will include it in a different PR